aws-opentelemetry-distro/src/amazon/opentelemetry/distro/_aws_metric_attribute_generator.py

```diff
@@ -445,6 +449,20 @@ def _set_remote_type_and_identifier(span: ReadableSpan, attributes: BoundedAttri
                 ":"
             )[-1]
             cloudformation_primary_identifier = _escape_delimiters(span.attributes.get(AWS_STEPFUNCTIONS_ACTIVITY_ARN))
+        elif is_key_present(span, AWS_LAMBDA_FUNCTION_NAME):
+            # To fix lambda topology issue, we handle the downstream lambda as a service ONLY IF the method
+            # call is "Invoke". Otherwise, we treat the downstream lambda as an AWS resource.
+            if span.attributes.get(_RPC_METHOD) == "Invoke":
+                attributes[AWS_REMOTE_SERVICE] = os.environ.get(
+                    "AWS_LAMBDA_REMOTE_SERVICE", span.attributes.get(AWS_LAMBDA_FUNCTION_NAME)
```

Comment:
Document AWS_LAMBDA_REMOTE_SERVICE and AWS_LAMBDA_REMOTE_ENVIRONMENT in somewhere, so, people know they are new for this code change.